tests/integration/profiles/five-safes-crate/test_5src_11_workflow_execution.py

```diff
@@ -0,0 +1,314 @@
+        }
+        WHERE {
+            ?this rdf:type schema:CreateAction ;
+                  schema:name ?name .
+            <./> schema:mentions ?this .
+        }
+        """
+
+    do_entity_test(
+        rocrate_path=ValidROC().five_safes_crate_request,
+        requirement_severity=Severity.REQUIRED,
+        expected_validation_result=False,
+        expected_triggered_requirements=None,
+        expected_triggered_issues=None,
+        profile_identifier="five-safes-crate",
+        rocrate_entity_mod_sparql=sparql,
+    )
+
+
+def test_5src_workflow_object_with_name_not_string():
+    sparql = """
+        PREFIX schema: <http://schema.org/>
+        PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+        DELETE {
+            ?this schema:name ?name .
+        }
+        INSERT {
+            ?this schema:name 123 .
+        }
+        WHERE {
+            ?this rdf:type schema:CreateAction ;
+                  schema:name ?name .
+            <./> schema:mentions ?this .
+        }
+        """
+
+    do_entity_test(
+        rocrate_path=ValidROC().five_safes_crate_request,
+        requirement_severity=Severity.REQUIRED,
+        expected_validation_result=False,
+        expected_triggered_requirements=None,
+        expected_triggered_issues=None,
+        profile_identifier="five-safes-crate",
+        rocrate_entity_mod_sparql=sparql,
+    )
+
+
+def test_5src_workflow_object_with_not_long_enough_name():
+    sparql = """
+        PREFIX schema: <http://schema.org/>
+        PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+        DELETE {
+            ?this schema:name ?name .
+        }
+        INSERT {
+            ?this schema:name "Too short" .
+        }
+        WHERE {
+            ?this rdf:type schema:CreateAction ;
+                  schema:name ?name .
+            <./> schema:mentions ?this .
+        }
+        """
+
+    do_entity_test(
+        rocrate_path=ValidROC().five_safes_crate_request,
+        requirement_severity=Severity.REQUIRED,
+        expected_validation_result=False,
+        expected_triggered_requirements=None,
+        expected_triggered_issues=None,
+        profile_identifier="five-safes-crate",
+        rocrate_entity_mod_sparql=sparql,
+    )
+
+
+def test_5src_workflow_object_has_no_start_time_if_begun():
+    sparql = """
+        PREFIX schema: <http://schema.org/>
+        PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+        DELETE {
+            ?s schema:startTime ?o .
+        }
+        WHERE {
+            ?s rdf:type schema:CreateAction;
+               schema:actionStatus ?status .
+               FILTER(?status IN (
+                    "http://schema.org/CompletedActionStatus",
+                    "http://schema.org/FailedActionStatus",
+                    "http://schema.org/ActiveActionStatus"
+                ))
+        }
+        """
+
+    do_entity_test(
+        rocrate_path=ValidROC().five_safes_crate_request,
```

Comment:
Doesn't matter for every test, but important for some, including this one: given the context of the workflow execution phase, these should target the result crate rather than the request. Right now this test does not trigger the desired issue because the request crate doesn't have any active or completed CreateActions

```suggestion
        rocrate_path=ValidROC().five_safes_crate_result,
```

Reply:
Yes, Thank you!!